<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>344</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>687</width>
    <height>344</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>687</width>
    <height>344</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AutomativePistonPlugin</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>372</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="inputLayout">
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="verticalSpacing">
      <number>6</number>
     </property>
     <item row="2" column="3">
      <widget class="QLabel" name="diametersLabel">
       <property name="text">
        <string>мм</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QComboBox" name="engineTypeComboBox">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <item>
        <property name="text">
         <string>Бензиновый</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Дизельный</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="diameterPinHoleLabel">
       <property name="text">
        <string>Диаметр отверстия для поршневого пальца (D):</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="pinHoleSize">
       <property name="text">
        <string>мм</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="engineTypeLabel">
       <property name="text">
        <string>Тип ДВС:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QComboBox" name="materialComboBox">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <item>
        <property name="text">
         <string>Чугун</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Алюминий</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="pistonHeadHeightLineEdit">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>16</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="materialLabel">
       <property name="text">
        <string>Материал поршня:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="pistonDiameterLineEdit">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>100</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="pistonHeightLineEdit">
       <property name="baseSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>55</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="pistonHeightLabel">
       <property name="text">
        <string>Высота поршня (H):</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="pistonHeadHeightLabel">
       <property name="text">
        <string>Высота головки поршня (H2):</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QPushButton" name="buildPistonButton">
       <property name="text">
        <string>Построить</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="heightHeadLabel">
       <property name="text">
        <string>мм</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="diametePinHoleLineEdit">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>18</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="heightLabel">
       <property name="text">
        <string>мм</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="pistonDiameterLabel">
       <property name="text">
        <string>Диаметр днища поршня (W):</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>291</width>
      <height>341</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="pistonSketch">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>291</width>
       <height>331</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/images/pistonsketch.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pistonHeightLineEdit</tabstop>
  <tabstop>pistonHeadHeightLineEdit</tabstop>
  <tabstop>pistonDiameterLineEdit</tabstop>
  <tabstop>diametePinHoleLineEdit</tabstop>
  <tabstop>engineTypeComboBox</tabstop>
  <tabstop>materialComboBox</tabstop>
  <tabstop>buildPistonButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
